### ISSUE TITLE
Better error message if wrong row,col is given

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1284]
```
gmt begin map pdf
gmt subplot begin 1x2 -Fs5c/5c
    gmt subplot set 1,1
    gmt basemap -R0/10/0/10 -Baf
    gmt subplot set 1,3
    gmt basemap -R0/10/0/10 -Baf
gmt subplot end
gmt end show
```

Currently, the error message is 
```
gmt [ERROR]: Unable to match specified row,col with subplot information in /Users/seisman/.gmt/sessions/gmt6.15040/gmt.subplot.0
subplot [ERROR]: Unable to match specified row,col with subplot information in /Users/seisman/.gmt/sessions/gmt6.15040/gmt.subplot.0
subplot [ERROR]: No subplot information file!
end [ERROR]: subplot was never completed - plot items in last panel may be missing
```

It would be better to say something like:
```
subplot col number not in allowed range 0-1.
```

**Comment by PaulWessel** [comment]
Yes, will fix this when tacking the zero-origin fix for row, col.  I added **-c**_index_ and gmt subplot set _index_ (starting at 1 for now) if you want to check it out.

**Comment by seisman** [comment]
Just tried this script:
```
gmt begin map pdf
gmt subplot begin 3x2 -Fs5c/5c
    gmt basemap -R0/10/0/10 -Baf -c1
    gmt basemap -R0/10/0/10 -Baf -c4
gmt subplot end
gmt end show
```

It can generate the figure correctly, but with some error messages:
```
gmt [ERROR]: Index 1 goes to (1, 1)
gmt [ERROR]: Index 4 goes to (2, 2)
psconvert [ERROR]: Could not create 'map'.pdf. Maybe input file does not fulfill PS specifications.
```

GMT version is 6.0.0_d654fdb_2019.08.01, the latest 6.0 branch.

**Comment by seisman** [comment]
Closed by #1394
